src/PayID/pay-id-client.ts

```diff
@@ -306,6 +306,7 @@ export default class PayIDClient {
   private static makeOptionsWithAcceptTypes(acceptType: string): object {
     const headers = {
       Accept: acceptType,
+      'PayID-Version': '1.0',
```

Comment:
Does the Key need to be quoted? 

Can you refactor 1.0 to a top level variable?

Reply:
> Can you refactor 1.0 to a top level variable?

Why would that be better?

Reply:
yeah, keys of js objects that have `-` need to be in quotes. if they have `_`, on the other hand, they don't

i'll do the top level var